documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,263 @@
+For each one of these effects, here's how the Navitia responses will be affected over the different endpoints. A link to the API
+
+
+## <a name="SIGNIFICANT_DELAYS"></a>Trip delayed
+
+![image](delay.png)
+
+``` shell
+# Extract of an impacted stop of /disruptions
+{
+    "amended_arrival_time": "194700",
+    "amended_departure_time": "194900",
+    "arrival_status": "delayed",
+    "base_arrival_time": "193200",
+    "base_departure_time": "193400",
+    "cause": "Panne d'un aiguillage",
+    "departure_status": "delayed",
+    "is_detour": false,
+    "stop_point": ⊕{7 items},
+    "stop_time_effect": "delayed",
+}
+```
+The effect of the disruption is `SIGNIFICANT_DELAYS`. It means that the train will arrive late at one or more stations in its journey.
+
+In the disruption, the delay can be found in the list of "impacted_stops" with the departure/arrival status set to "delayed". See the [disruption](#disruption) objects section for its full content and description.
+
+<div></div>
+### Journeys
+
+``` shell
+# Request example for /journeys
+http://api.navitia.io/v1/coverage/<coverage>/journeys?from=<origin>&to=<destination>&data_freshness=realtime
+```
+
+``` shell
+# Extract of the public transport section of the response /journeys
+
+    "sections": ⊖[
+       ⊖{
+            "additional_informations": ⊕[1 item],
+            "arrival_date_time": "20190529T205000",
+            "base_arrival_date_time": "20190529T204500",
+            "base_departure_date_time": "20190529T160000",
+            "co2_emission": ⊕{2 items},
+            "data_freshness": "realtime",
+            "departure_date_time": "20190529T160000",
+            "display_informations": ⊕{13 items},
+            "duration": 21180,
+            "from": ⊕{5 items},
+            "geojson": ⊕{3 items},
+            "id": "section_0_0",
+            "links": ⊕[6 items],
+            "stop_date_times": ⊖[
+               ⊕{7 items},
+               ⊖{
+                    "additional_informations": [],
+                    "arrival_date_time": "20190605T194700",
+                    "base_arrival_date_time": "20190605T193200",
+                    "base_departure_date_time": "20190605T193400"
+                    "departure_date_time": "20190605T194900",
+                    "links": [],
+                    "stop_point": ⊕{7 items},
+                },
+               ⊕{7 items},
+               ⊕{7 items},
+            ]
+            "to": ⊕{5 items},
+            "type": "public_transport",
+        }
+    ]
+```
+
+
+The status of the journey is `SIGNIFICANT_DELAYS`. In a public transport section of the response, "base_arrival_date_time"/"base_departure_date_time" represent the scheduled arrival/departure time without taking into account the delay whereas "arrival_date_time"/"departure_date_time" are the actual arrival/departure time, after the delay is applied. The delay can also be observed for every stop_point of the journey with the same parameters in "stop_date_times".
```

Comment:
"stop_point" is a value from the code so it should be between comma ?

Reply:
Why? Why not? Let's call a referee!

Reply:
No big preference, but I'd vote to let it as-is.
To me, it's a transportation object in that context.
Maybe we can write `stop point` or `stop-point` to show that it's kind of real-life object, but the current way is probably the more consistent with the rest of the doc.

Reply:
:checkered_flag: `stop point` everywhere :champagne: